README.md

```diff
@@ -15,6 +15,7 @@ Each package name links to the documentation for that package.
 
 | Package | Latest version | Description |
 |---------|----------------|-------------|
+| [Google.Analytics.Admin.V1Alpha](https://googleapis.dev/dotnet/Google.Analytics.Admin.V1Alpha/1.0.0-alpha01) | 1.0.0-alpha01 | Analytics Admin API |
```

Comment:
For later, when we have more, but it might not be a bad idea to have separate tables?

Reply:
Between Cloud and non-Cloud? Maybe, yes.

Reply:
Yes, between Cloud and non-Cloud.